src/controllers/commentsController.js

```diff
@@ -0,0 +1,169 @@
+import commentQueries from '../models/queries/commentQueries';
+import ArticleQueries from '../models/queries/articleQueries';
+
+// eslint-disable-next-line require-jsdoc
+class Comment {
+  /**
+   * Adds two numbers together.
+   * @param { object } req the request.
+   * @param { object } res The response.
+   * @returns { object } the return object.
+   */
```

Comment:
Function `createComment` has 27 lines of code (exceeds 25 allowed)

Reply:
Yeap. Let me change..